server/validation/event.js

```diff
@@ -9,7 +9,10 @@ export default {
       placeId: Joi.number().required(),
       resources: Joi.array().items(Joi.number()).required(),
       slots: Joi.array().min(1).unique().items(Joi.number()).required(),
+      socialMedia: Joi.string().min(3),
+      ticketUrl: Joi.string().min(3),
```

Comment:
The Joi validation library should have a method to check for valid URLs, I would recommend using that here.

Reply:
I added url validation here https://github.com/adzialocha/hoffnung3000/pull/92/commits/29cedad82e1f15e64793dbf0f47ffeb4a0032049 and to help users input in the correct format I set the initialValue to `https://`. To avoid saving this value to the server when no address is input (it's optional) I'm checking the `ticketUrl:` and `websiteUrl:` for `'https://'` and setting as `undefined` if `true`. Is this unconventional, or Ok?

https://github.com/adzialocha/hoffnung3000/blob/29cedad82e1f15e64793dbf0f47ffeb4a0032049/app/scripts/views/EventsEdit.js#L49-L53